### PR TITLE
Alexis s/video route

### PR DESCRIPTION
**Pull request by Alexis559**

### CODE REVIEW

File: src/routes/video.route.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const M = require('../models');
+const logger = require('../helpers/logger');
+
+module.exports = (router) => {
+  router.post('/video', async (req, res) => {
```
Comment (yannick-mayeur) [inline src/routes/video.route.js]: `video` should be plural

File: src/routes/video.route.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    logger.log('info', 'received request: POST /video\nbody:', req.body);
+    M.Video.create(req.body)
+      .then((video) => res.status(200).send(video))
+      .catch((err) => {
+        logger.log('error', 'POST /video failed', err);
+        res.sendStatus(500);
+      });
+  });
+
+  router.get('/video/getByName', async (req, res) => {
```
Comment (yannick-mayeur) [inline src/routes/video.route.js]: you should pass parameters for a get request like so: https://expressjs.com/en/guide/routing.html#route-parameters

File: src/routes/video.route.js (hunk 1 above)
Comment (Alexis559) [reply src/routes/video.route.js]: Done.

**Comment by yannick-mayeur** [comment]
Why do we need a route to get a video by name ?

**Pull request merged**